app/views/declaration/additional_information.scala.html

```diff
@@ -20,14 +20,15 @@
 @import forms.declaration.AdditionalInformation
 @import uk.gov.hmrc.play.views.html._
 
-@(appConfig: AppConfig, form: Form[AdditionalInformation], items: Seq[AdditionalInformation])(implicit request: Request[_], messages: Messages)
+@import forms.declaration.CommodityMeasure
+@(itemId: String, appConfig: AppConfig, form: Form[AdditionalInformation], items: Seq[AdditionalInformation])(implicit request: Request[_], messages: Messages)
 
 @main_template(
     title = messages("supplementary.additionalInformation"),
     appConfig = appConfig
 ) {
 
-    @components.back_link("/customs-declare-exports/declaration/commodity-measure")
+    @components.back_link(CommodityMeasureController.displayPage(itemId).url)
```

Comment:
👍

Reply:
I think IntelliJ struggles to compile it unless you stick `routes.` as a prefix e.g. `routes.CommodityMeasureController.displayPage(...)`

I think it gets confused wether you want the `controllers.CommodityMeasureController` or the `routes.CommodityMeasureController`.  

All works fine though as it is, ^^ just gets rid of all the red in IntelliJ

Reply:
This is not just my changes though, seen this elsewhere.. another ticket for the dev board?